web-ng/src/app/components/color-picker/color-picker.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, ElementRef, OnInit, Inject, Output, EventEmitter } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { MatDialog, MatDialogConfig } from "@angular/material";
+import { ColorEvent } from 'ngx-color';
+
+@Component({
+  selector: 'app-color-picker',
+  templateUrl: './color-picker.component.html',
+  styleUrls: ['./color-picker.component.css']
+})
+export class ColorPickerComponent implements OnInit {
+
+  private readonly _matDialogRef: MatDialogRef<ColorPickerComponent>;
+  private readonly triggerElementRef: ElementRef;
+  @Output() public onColorPicked: EventEmitter<{}> = new EventEmitter();
+  constructor(private dialog: MatDialog, _matDialogRef: MatDialogRef<ColorPickerComponent>,
```

Comment:
Please remove "_" before private member.

Reply:
Fixed it thanks :)